gradle/libs.versions.toml

```diff
@@ -79,7 +79,7 @@ scala-collection-compat = "2.13.0"
 slf4j = "2.0.16"
 snowflake-jdbc = "3.22.0"
 spark-hive34 = "3.4.4"
-spark-hive35 = "3.5.4"
+spark-hive35 = "3.5.5"
```

Comment:
a silly question: why call it `spark-hive3x` instead of `spark3x`? a quite weird name

Reply:
I guess it's because we depend on spark-hive module